bowtie/_cli.py

```diff
@@ -154,31 +163,63 @@ def summary(input: Iterable[str], format: str | None):
         key=lambda each: (sum(each[1].values()), each[0][0]),  # type: ignore[reportUnknownLambdaType]  # noqa: E501
         reverse=True,
     )
+    validity = {
+        str(summary._combined[schemas]["case"]["schema"]["type"]): [
```

Comment:
This is accessing a `type` property in the schemas which were tested -- I'm not sure what you intend by doing so -- a schema may not have a `type` keyword in it, and if it did, it may not be unique across all schemas that were tested -- someone may have used `{"type": "integer", "minimum": 7}` and some other schema says `{"type": "integer", "title": "my cool schema"}` -- but the data structures can get complicated to think about here.

You should likely be trying to build up a list rather than a dictionary -- one ordered by the order we evaluated schemas here, so no need for a key. It probably also is worth thinking about splitting this function in two now -- `combined` here as a list is meant to generate the table used for `--show failures`, and it's that that you're building an alternative to here in order to collect validation results. As I say, after the weekend I can try to provide more specific hints.